<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="urLabel">
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="usrLineEdit">
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="macLabel">
     <property name="text">
      <string>选择网卡地址：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="macBoBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ipLabel">
     <property name="text">
      <string>选择ip地址：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="ipBoBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="portLabel">
     <property name="text">
      <string>端口:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="portSpinBox">
     <property name="minimum">
      <number>1024</number>
     </property>
     <property name="maximum">
      <number>50000</number>
     </property>
     <property name="value">
      <number>11451</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="settingBtn">
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
